src/components/Blocking/SpecificIssue.js

```diff
@@ -89,7 +89,7 @@ const styles = StyleSheet.create({
     color: COLORS.black
   },
   input: {
-    backgroundColor: COLORS.lightgrey,
+    backgroundColor: COLORS.gray100,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The background color has been updated from `COLORS.lightgrey` to `COLORS.gray100`. Make sure this new color aligns with the overall theme and doesn't affect the readability of any overlaid text or UI elements.
```diff
-    backgroundColor: COLORS.lightgrey
+    backgroundColor: COLORS.gray100
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->